psi4/src/psi4/libscf_solver/sad.cc

```diff
@@ -390,14 +390,19 @@ void SADGuess::get_uhf_atomic_density(std::shared_ptr<BasisSet> bas, std::shared
             throw PSIEXCEPTION("SAD: Fractional occupations are not supported beyond Radon");
         }
 
-        nalpha = nfzc + nact;
-        nbeta = nalpha;
-        double frac_act = std::pow(((double)(Z - nfzc * 2)) / ((double)nact * 2), 0.5);
+        // Number of occupied orbitals is
+        int norb(nfzc+nact);
```

Comment:
Please do not initialize ints this way, use `int norb = ...` to conform to normal Psi4 syntax.